.vimrc

```diff
@@ -84,8 +84,10 @@ set cursorline
 
 ""------------ Mappings --------------"
 "" Create tree navigation
-map <C-n> :vsp . <bar> vertical resize 30 <cr>
+map <C-n> :Lexplore <CR>
```

Comment:
Dude, how about keeps this part in the same way? Think about that, the old style works in every kind of vim. I believe this is a better option. However, if you disagree, how about to talk in google hangout or if it is possible talk personally?